security-release-process-documentation/security-release-process.md

```diff
@@ -270,16 +270,16 @@ after joining the list then you will be unsubscribed.
 
 ### Requesting to Join
 
-New membership requests are sent to kubernetes-security@googlegroups.com.
+New membership requests are sent to security@kubernetes.io
 
 In the body of your request please specify how you qualify and fulfill each
 criterion listed in [Membership Criteria](#membership-criteria).
```

Comment:
Can we have a target response time and an escalation policy for blackholes?

Reply:
@justinsb that is reasonable feedback but I won't address it in this PR. Please use the discussion mailing list: https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#contacting-the-team